pkgs/by-name/fi/ficsit-cli/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "ficsit-cli";
+  version = "0.6.0";
+  commit = "5dc8bdbaf6e8d9b1bcd2895e389d9d072d454e15";
+
+  src = fetchFromGitHub {
+    owner = "satisfactorymodding";
+    repo = "ficsit-cli";
+    tag = "v${version}";
+    hash = "sha256-Zwidx0war3hos9NEmk9dEzPBgDGdUtWvZb7FIF5OZMA=";
+  };
+
+  ldflags = [
+    "-X=main.version=v${version}"
+    "-X=main.commit=${commit}"
+  ];
+
+  doCheck = false;
+
+  vendorHash = "sha256-vmA3jvxOLRYj5BmvWMhSEnCTEoe8BLm8lpm2kruIEv4=";
+
+  meta = {
+    description = "CLI tool for managing mods for the game Satisfactory ";
+    homepage = "https://github.com/satisfactorymodding/ficsit-cli";
+    license = lib.licenses.gpl3Only;
+    platforms = [ "x86_64-linux" ];
```

Comment:
The CLI is designed to work on any platform, as it has remote configuration capabilities. So it should be left empty.

Reply:
Removed